examples/hello_worlds.rb

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require 'vedeu'
```

Comment:
On 01.10.2015 17:10, Gavin Laking wrote:
> In examples/hello_worlds.rb
> <https://github.com/gavinlaking/vedeu/pull/244#discussion_r40924458>:
> 
>> @@ -0,0 +1,77 @@
>> +#!/usr/bin/env ruby
>> +
>> +require 'vedeu'
> 
> Sorry, I'm not sure if I've miscommunicated here. The |fail
> Vedeu::VERSION| line was simply to allow you to compare the versions of
> Vedeu.

Yes, but for my points that doesnt make a different (if it fails before
anything, it failed through the require `'bundler/setup'`.

> I've just built a Docker image which will allow me to test in a
> 'bare-bones' fashion (who knows what crazy workarounds are in my
> development environment! :sweat_smile:.

Well, you could also go with rvm and gemsets.
Have fun and thanks for looking into all that
--felix